### ISSUE TITLE
Tool not found

### ISSUE THREAD

**Issue by CraigB7708** [MicrosoftDocs/SCCMdocs#1170]
Not sure if this needs updating but on my 1802 version of SCCM this tool does not exist

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0bc237e7-4eee-13e3-01ca-de94df0c9a0c
* Version Independent ID: 7dd0be7d-50c7-6a22-d247-7f192829528b
* Content: [Deployment Monitoring Tool - Configuration Manager](https://docs.microsoft.com/en-us/sccm/core/support/deployment-monitoring-tool)
* Content Source: [sccm/core/support/deployment-monitoring-tool.md](https://github.com/Microsoft/SCCMdocs/blob/master/sccm/core/support/deployment-monitoring-tool.md)
* Product: **configuration-manager**
* GitHub Login: @aczechowski
* Microsoft Alias: **aaroncz**

**Comment by jocasti** [comment]
You're right. According to this doc, these tools were introduced in v1806. https://docs.microsoft.com/en-us/sccm/core/support/tools
